packages/amplication-data-service-generator-runner/project.json

```diff
@@ -58,8 +58,9 @@
       },
       "dependsOn": ["generate-example-input-json"]
     },
-    "docker:build": {
+    "docker-build": {
```

Comment:
leave docker:build since it will benefit from the default dependencies like `pre build`, `build` and `postbuild` @tupe12334 @abrl91

Reply:
this already was fixed